### ISSUE TITLE
[a11y] Add aria-label prop in SfCircleIcon stories

### ISSUE THREAD

**Issue by aniamusial** [DivanteLtd/storefront-ui#722]
**Describe the bug**
There is missing aria-label for SfCircleIcon button.

**How to reproduce**
1. Go to Storybook and choose SfCircleIcon
2. Click on 'Accessability' tab -> Violations
3. See the error

**Proposed solution**
Add aria label prop in stories and assign it. Or we can think if we shouldn't set default ariaLabel different than ""?
